usaspending_api/awards/v2/views/awards.py

```diff
@@ -1,12 +1,17 @@
-from rest_framework.response import Response
-from rest_framework.request import Request
+import logging
+
 from django.db.models import Max
+from rest_framework.request import Request
+from rest_framework.response import Response
 
 from usaspending_api.awards.models import Award
+from usaspending_api.awards.serializers_v2.serializers import AwardContractSerializerV2, AwardMiscSerializerV2
 from usaspending_api.common.cache_decorator import cache_response
+from usaspending_api.common.exceptions import NotImplementedException, UnprocessableEntityException
 from usaspending_api.common.views import APIDocumentationView
 from usaspending_api.core.validator.tinyshield import TinyShield
-from usaspending_api.awards.serializers_v2.serializers import AwardContractSerializerV2, AwardMiscSerializerV2
+
+logger = logging.getLogger("console")
```

Comment:
The file has imports `logging`  and declares `logger`, which I would imagine would be for line 62, but since that line and the logger usage is removed, does the script still require logging?